Packages/MIES/MIES_Utilities.ipf

```diff
@@ -6826,3 +6826,35 @@ Function/WAVE ZapNullRefs(WAVE/WAVE input)
 
 	return result
 End
+
+/// @brief Flatten multidimensional waves inside input to the given dimension
+///
+/// @param input wave reference wave
+/// @param sdim  [optional, defaults to 1] dimensionality to flatten to
+Function/WAVE FlattenWaves(WAVE/WAVE input, [variable sdim])
```

Comment:
I think the function name is very generic and should be more specific.

Also with "flatten" I would expect that {{1,2},{3,4} -> {1,2,3,4} (like Redimension/E=1) but it does actually {1,2} and {3,4}.

How about something like "SplitWavesToDimension" ?

Reply:
Done.